DTIPrep/DTIPrepRegister.pl

```diff
@@ -206,7 +216,7 @@
                                                     $DTIPrepVersion);
 if (!$registeredXMLReportFile) {
     print LOG "\nERROR: no XML report file was registered in the database\n";
-    exit 0;
+    exit $NeuroDB::ExitCodes::XML_QCREPORT_INSERT_FAILURE;
```

Comment:
also, why not use the code `$FILE_REGISTRATION_FAILURE`?

Reply:
Good point! Will be changing those.